_posts/patterns/2015-03-25-headings.md

```diff
@@ -3,7 +3,7 @@ layout:         pattern
 title:          Headings
```

Comment:
I would really love to see that happen too.  I find the date unnecessary as well.  If we wanted to keep the dates in, we could add them to the post's front matter.